### PR TITLE
feat(components/login/passwordReset): Create scaffolding for passwordReset component

### PR DESCRIPTION
**Pull request by oegea**
From CNET PRO, we are working to implement a generic password reset component that will fully render the interface to execute the password recovery flow.

This PR starts creating the scaffolding.

### CODE REVIEW

File: components/login/passwordReset/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# LoginPasswordReset
+
+Work in progress.
```
Comment (jordevo) [inline components/login/passwordReset/README.md:3]: 🍆 something we could already add at this point, right?

**Review by davidmartin84 — APPROVED** [review]

File: components/login/passwordReset/README.md (hunk 1 above)
Comment (oegea) [reply components/login/passwordReset/README.md:3]: We'll add ASAP 😅 
As we'll be working on this on the following days, a lot of Prs will be submitted to add features and complete the component. This is just a fast scaffolding to ensure we share the same component structure.

File: components/login/passwordReset/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import '~@s-ui/theme/lib/index';
+
+.sui-LoginPasswordReset {
```
Comment (jordevo) [inline components/login/passwordReset/src/index.scss:3]:
```suggestion
.mt-LoginPasswordReset {
```

🍆  should be like this, right?

**Review by jordevo — APPROVED** [review]

File: components/login/passwordReset/src/index.scss (hunk 2 above)
Comment (davidmartin84) [reply components/login/passwordReset/src/index.scss:3]: This component is intended to be global. reusable for all verticals. Maybe we can think if the prefix "sui-" is the best... but coupling it to "mt" is not the idea.

Comment (jordevo) [reply components/login/passwordReset/src/index.scss:3]: you're absolutely right! I missed we're on this repo! 😅

**Pull request merged**